### PR TITLE
APP-2089: add new constants for recurrence notification

### PR DESCRIPTION
**Pull request by railianmaksym**

### CODE REVIEW

**Comment by gilbertparreno** [comment]
create helper methods too for recurrence object events.

**Comment by eunicegalang** [comment]
Upgrade `version` from ApplicationConfiguration

File: wallet/src/main/java/com/paysera/lib/wallet/entities/notification/NotificationSubscriber.java [hunk 1]
```diff
@@ -228,4 +228,28 @@ public NotificationEvent getCardTransactionSuccessfulEvent() {
...
+                return event;
+            }
+        }
+        return null;
+    }
+
+    public NotificationEvent getRecurrenceTransferFailedEvent() {
+        for (NotificationEvent event : events) {
+            if (
+                    event.eventName.equals(NotificationEvent.EVENT_NAME_FAILED)
```
Comment (eunicegalang) [inline wallet/src/main/java/com/paysera/lib/wallet/entities/notification/NotificationSubscriber.java:247]: fix spacing indentation

File: wallet/src/main/java/com/paysera/lib/wallet/entities/notification/NotificationSubscriber.java [hunk 2]
```diff
@@ -228,4 +228,28 @@ public NotificationEvent getCardTransactionSuccessfulEvent() {
         }
         return null;
     }
-}
+
+    public NotificationEvent getRecurrenceTransferDoneEvent() {
+        for (NotificationEvent event : events) {
+            if (
+                    event.eventName.equals(NotificationEvent.EVENT_NAME_DONE)
```
Comment (eunicegalang) [inline wallet/src/main/java/com/paysera/lib/wallet/entities/notification/NotificationSubscriber.java:235]: here too

**Review by eunicegalang — CHANGES_REQUESTED** [review]

File: wallet/src/main/java/com/paysera/lib/wallet/entities/notification/NotificationSubscriber.java (hunk 1 above)
Comment (railianmaksym) [reply wallet/src/main/java/com/paysera/lib/wallet/entities/notification/NotificationSubscriber.java]: ✅

File: wallet/src/main/java/com/paysera/lib/wallet/entities/notification/NotificationSubscriber.java (hunk 2 above)
Comment (railianmaksym) [reply wallet/src/main/java/com/paysera/lib/wallet/entities/notification/NotificationSubscriber.java]: ✅

**Comment by gilbertparreno** [comment]
everything else is okay with me :)

**Pull request merged**